src/main/java/com/webonise/login/controller/SignUpController.java

```diff
@@ -1,23 +1,40 @@
 package com.webonise.login.controller;
 
-import com.webonise.login.model.UserDTO;
-import com.webonise.login.model.UserRequest;
-import com.webonise.login.service.UserService;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.webonise.login.model.UserDTO;
+import com.webonise.login.model.UserEntity;
+import com.webonise.login.model.UserRequest;
+import com.webonise.login.service.UserService;
+
 @RestController("/user")
 @RequestMapping("/")
```

Comment:
what is the difference between @RestController, @RequestMapping what if I want to use only one Annotation